crates/bevy_sprite_render/src/lib.rs

```diff
@@ -0,0 +1,333 @@
+pub type SpriteSystem = SpriteSystems;
+
+impl Plugin for SpriteRenderingPlugin {
+    fn build(&self, app: &mut App) {
+        load_shader_library!(app, "render/sprite_view_bindings.wgsl");
+
+        embedded_asset!(app, "render/sprite.wgsl");
+
+        if !app.is_plugin_added::<TextureAtlasPlugin>() {
+            app.add_plugins(TextureAtlasPlugin);
+        }
+
+        app.add_plugins((
+            Mesh2dRenderPlugin,
+            ColorMaterialPlugin,
+            TilemapChunkPlugin,
+            TilemapChunkMaterialPlugin,
+        ))
+        .add_systems(
+            PostUpdate,
+            (
+                calculate_bounds_2d.in_set(VisibilitySystems::CalculateBounds),
+                (
+                    compute_slices_on_asset_event.before(AssetEventSystems),
+                    compute_slices_on_sprite_change,
+                )
+                    .in_set(SpriteSystems::ComputeSlices),
+            ),
+        );
+
+        app.register_required_components::<Sprite, SyncToRenderWorld>();
+
+        if let Some(render_app) = app.get_sub_app_mut(RenderApp) {
+            render_app
+                .init_resource::<ImageBindGroups>()
+                .init_resource::<SpecializedRenderPipelines<SpritePipeline>>()
+                .init_resource::<SpriteMeta>()
+                .init_resource::<ExtractedSprites>()
+                .init_resource::<ExtractedSlices>()
+                .init_resource::<SpriteAssetEvents>()
+                .init_resource::<SpriteBatches>()
+                .add_render_command::<Transparent2d, DrawSprite>()
+                .add_systems(RenderStartup, init_sprite_pipeline)
+                .add_systems(
+                    ExtractSchedule,
+                    (
+                        extract_sprites.in_set(SpriteSystems::ExtractSprites),
+                        extract_sprite_events,
+                    ),
+                )
+                .add_systems(
+                    Render,
+                    (
+                        queue_sprites
+                            .in_set(RenderSystems::Queue)
+                            .ambiguous_with(queue_material2d_meshes::<ColorMaterial>),
+                        prepare_sprite_image_bind_groups.in_set(RenderSystems::PrepareBindGroups),
+                        prepare_sprite_view_bind_groups.in_set(RenderSystems::PrepareBindGroups),
+                        sort_binned_render_phase::<Opaque2d>.in_set(RenderSystems::PhaseSort),
+                        sort_binned_render_phase::<AlphaMask2d>.in_set(RenderSystems::PhaseSort),
+                    ),
+                );
+        };
+    }
+}
+
+/// System calculating and inserting an [`Aabb`] component to entities with either:
+/// - a `Mesh2d` component,
+/// - a `Sprite` and `Handle<Image>` components,
+///   and without a [`NoFrustumCulling`] component.
+///
+/// Used in system set [`VisibilitySystems::CalculateBounds`].
+pub fn calculate_bounds_2d(
```

Comment:
I believe this system can stay in bevy_sprite, but I don't feel very strongly about it.

Reply:
Actually, there's no reason it shouldnt be there, since it wouldnt need any additional imports to live there

Reply:
(it depends on bevy_camera, and bevy_camera depends on bevy_mesh)